docs/source/glm.rst

```diff
@@ -180,6 +180,7 @@ available link functions can be obtained by
    CDFLink
    CLogLog
    LogLog
+   CLog
```

Comment:
LogC

Reply:
Thanks for finding this. It is fixed now.